alpine/distributionscanner.go

```diff
@@ -34,36 +34,44 @@ type alpineRegex struct {
 // ex: "Welcome to Alpine Linux 3.3"
 var alpineRegexes = []alpineRegex{
 	{
-		release: V3_3,
-		regexp:  regexp.MustCompile(`Alpine Linux (v)?3.3`),
+		dist:   alpine3_12Dist,
+		regexp: regexp.MustCompile(`Alpine Linux v?3\.12`),
```

Comment:
by changing all these regex's in this pr, you are confirming you tested each and every regexp match here, right?